closed/test/jdk/openj9/internal/security/property-java.security

```diff
@@ -0,0 +1,531 @@
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+ * ===========================================================================
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * IBM designates this particular file as subject to the "Classpath" exception
+ * as provided by IBM in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * ===========================================================================
+ */
```

Comment:
I don't expect this comment is legal in a properties file: use `#` like below.

Reply:
@keithc-ca 
Do we need to add the copyright info in this test case used java.security file? I checked the java.security file under the java.base which we added those restricted security profiles, there is no copyright info.

Reply:
Yes, I think this file should have a copyright notice; it just needs to use the right comment syntax.

Reply:
Done.